### PR TITLE
Update changelog for 0.32.0rc1

### PR DESCRIPTION
**Pull request by sklam**
as titled.

### CODE REVIEW

File: CHANGE_LOG [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
+  platforms.
+* PR #551: Remove python 2, python < 3.6, fix up, add flake8
+* PR #549: Miscalled method and missing parameter in the documentation
+* PR #547: Permit building on Visual Studio 2017
+* PR #543: Update error message in LLVM version check.
+* PR #540: update to final release date for 0.31.0
+
+Authors:
+
+* ARF1
```
Comment (sklam) [inline CHANGE_LOG]:
```suggestion
* Arik Funke
```

File: CHANGE_LOG [hunk 2]
```diff
@@ -1,7 +1,34 @@
-v0.32.0dev
+v0.32.0rc1
```
Comment (stuartarchibald) [inline CHANGE_LOG]: Do we normally at the data at RC phase? If we don't add the date now will we remember to do it later?

File: CHANGE_LOG [hunk 3]
```diff
@@ -1,7 +1,34 @@
...
 ----------
 
-In development
+The main changes in this release is the removal of specific code for Python 2
+and Python <3.6.
```
Comment (stuartarchibald) [inline CHANGE_LOG]: Code base is also PEP8 compliant.

Comment (sklam) [reply CHANGE_LOG]: i'll add that

File: CHANGE_LOG [hunk 4]
```diff
@@ -1,7 +1,34 @@
-v0.32.0dev
-----------
+v0.32.0 (Apr 9, 2020)
+---------------------
+
+The main changes in this release is the removal of specific code for Python 2
+and Python <3.6, and making the code base is PEP8 compliant.
```
Comment (stuartarchibald) [inline CHANGE_LOG]:
```suggestion
and Python <3.6, and making the code base PEP8 compliant.
```

File: CHANGE_LOG [hunk 5]
```diff
@@ -1,7 +1,34 @@
-v0.32.0dev
-----------
+v0.32.0 (Apr 9, 2020)
+---------------------
+
+The main changes in this release is the removal of specific code for Python 2
```
Comment (stuartarchibald) [inline CHANGE_LOG]:
```suggestion
The main changes in this release are the removal of specific code for Python 2
```

**Pull request merged**